packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -0,0 +1,74 @@
+import { getL2Network, getL1Network } from '@arbitrum/sdk'
+import { useWallet } from '@arbitrum/use-wallet'
+import { Popover, Transition } from '@headlessui/react'
+import { useAppState } from '../../state'
+import { ChainId, getNetworkName } from '../../util/networks'
+import {
+  changeNetworkBasic,
+  networkStyleMap,
+  networkSelectionList
+} from '../../util/NetworkUtils'
+
+export const NetworkSelectionContainer = ({
+  children,
+  filterNetworks
+}: {
+  children: React.ReactNode
+  filterNetworks?: (networkId: ChainId) => boolean
```

Comment:
`filterNetworks` was not passed in when used in the codebase, do we foresee it to be used in the future? Or else can we remove it?

Reply:
bump^